test/tests/globalparams/global_param/tests

```diff
@@ -1,12 +1,28 @@
 [Tests]
   design = 'Parser.md'
-  issues = '#437'
+  issues = '#437 #8761'
 
-  [./global_param_test]
+  [global_param_test]
     type = 'Exodiff'
     input = 'global_param_test.i'
     exodiff = 'out.e'
 
     requirement = 'The system shall include an input file syntax that supports defining global options.'
-  [../]
+  []
+  
+  [test_suppress_ignore]
+    type = 'RunException'
+    input = 'suppresscheck.i'
```

Comment:
rename this file to suppress_check.i 
it s the convention for all test inputs names